.github/workflows/release.yaml

```diff
@@ -85,4 +85,12 @@ jobs:
         args: release --rm-dist --timeout 60m --skip-validate --config=./.goreleaser.yaml --release-notes=${{ env.RELEASE_NOTES_FILE }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GORELEASER_CURRENT_TAG: ${{ env.RELEASE_VERSION }}
\ No newline at end of file
+        GORELEASER_CURRENT_TAG: ${{ env.RELEASE_VERSION }}
+    - name: Repository Dispatch
+      if: inputs.release_candidate == false
+      uses: peter-evans/repository-dispatch@v2
+      with:
+        token: ${{ secrets.EVENT_API }}
```

Comment:
TODO: Add an action secret with a token (cc @In-Ko)

Reply:
Let's mark the PR as draft until this token is available.

Reply:
I have create the token and secret now @nikimanoledaki